protocols/virtualfund/virtualfund_single_hop_test.go

```diff
@@ -172,26 +172,29 @@ func TestClone(t *testing.T) {
 	}
 }
 
-func collectPeerSignaturesOnSetupState(V *channel.SingleHopVirtualChannel, myRole uint, prefund bool) {
+func collectPeerSignaturesOnSetupState(v *channel.SingleHopVirtualChannel, myRole uint, prefund bool) *channel.SingleHopVirtualChannel {
```

Comment:
Makes the non-modifying intent explicit.

```suggestion
func collectPeerSignaturesOnSetupState(v channel.SingleHopVirtualChannel, myRole uint, prefund bool) *channel.SingleHopVirtualChannel {
```

Reply:
Done in a9a0dde.